### PR TITLE
Fixes for javadocs

### PR DESCRIPTION
**Pull request by tildar**
Building the javadoc is a warning mess, this should fix it.

### CODE REVIEW

**Comment by tildar** [comment]
Hmm... Seems I broke something. I'll fix it soon.

File: code/src/java/pcgen/rules/context/RuntimeReferenceContext.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 import pcgen.cdom.reference.ManufacturableFactory;
 import pcgen.cdom.reference.ReferenceManufacturer;
 import pcgen.cdom.reference.SimpleReferenceManufacturer;
-import pcgen.persistence.PersistenceLayerException;
+// import pcgen.persistence.PersistenceLayerException;
```
Comment (karianna) [inline code/src/java/pcgen/rules/context/RuntimeReferenceContext.java]: why commented out?

File: code/src/java/plugin/encounter/EncounterPlugin.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
 import java.io.File;
-import java.io.FileNotFoundException;
+// import java.io.FileNotFoundException;
```
Comment (karianna) [inline code/src/java/plugin/encounter/EncounterPlugin.java]: same here

File: code/src/java/pcgen/rules/context/RuntimeReferenceContext.java (hunk 1 above)
Comment (tildar) [reply code/src/java/pcgen/rules/context/RuntimeReferenceContext.java]: One of the tests flagged it as unused. I didn't want to just delete it. so I commented it.

File: code/src/java/plugin/encounter/EncounterPlugin.java (hunk 2 above)
Comment (tildar) [reply code/src/java/plugin/encounter/EncounterPlugin.java]: One of the tests flagged it as unused. I didn't want to just delete it. so I commented it.

Comment (tildar) [reply code/src/java/plugin/encounter/EncounterPlugin.java]: Looking back I see 2 PMD rule violations were found by travis, thus it failed.  Commenting these fixed it.

Comment (tildar) [reply code/src/java/plugin/encounter/EncounterPlugin.java]: In both cases I see I had in a prior commit removed an @throw when there was no throw to be found. Thus the imports became unused and travis complained.

Comment (karianna) [reply code/src/java/plugin/encounter/EncounterPlugin.java]: Just remove them in that case :-)

File: code/src/java/pcgen/rules/context/RuntimeReferenceContext.java (hunk 1 above)
Comment (karianna) [reply code/src/java/pcgen/rules/context/RuntimeReferenceContext.java]: It's safe to remove :-)

**Comment by karianna** [comment]
@tildar one codacy bug to fix

**Pull request merged**